assets/src/components/v2/pre_fare/no_data.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ComponentType } from "react";
+import { default as LcdNoData } from "Components/v2/lcd/no_data";
```

Comment:
When importing a module's default export, you can name it whatever you want.
```suggestion
import LcdNoData from "Components/v2/lcd/no_data";
```

Reply:
Not sure why I blanked on this. In my mind, the name of the import had to match the export 😬 Got that fixed.